### DISCUSSION TITLE
Contributing to executablebooks

### DISCUSSION THREAD

**Discussion by awvwgk** [executablebooks/meta#577] [General]
First, thanks for developing and maintaining all those great projects. I've been a user of several EB projects for a while and only recently managed to break them and would like to contribute back now.

So far I only saw a bot welcoming me several times, but it is no good at answering question, therefore I'm posting to get some information on how to get involved and contribute patches.

**Comment by chrisjsewell** [comment]
Hey @awvwgk, yep I saw your comment on sphinx-design thanks 👍 was just finishing up the release of myst-parser, then was planning to circle back round to sphinx-design 😄
